src/Parser/VengeanceDemonHunter/CombatLogParser.js

```diff
@@ -127,7 +128,13 @@ class CombatLogParser extends MainCombatLogParser {
 
     results.statistics = [
       <StatisticBox
-        icon={<Icon icon="class_demonhunter" alt="Damage taken" />}
+        icon={<Icon icon="class_demonhunter" alt="Damage done" />}
+        value={`${formatNumber(this.totalDamageDone / this.fightDuration * 1000)} DPS`}
+        label='Damage done'
```

Comment:
I'm no fan of having tanks focus on damage done so I'd prefer showing DT first as a way of suggesting it is more important, but I guess these statistics for tanks will need some more refining so you can leave it as is and we'll change it when we figure out the best way to show it that can be consistent among all tanks.